contracts/BosonRouter.sol

```diff
@@ -392,16 +430,16 @@ contract BosonRouter is
         bytes32 rDeposit,
         bytes32 sDeposit // tokenDeposits
     ) external override nonReentrant whenNotPaused {
-        (uint256 price, uint256 depositBu) =
-            IVoucherKernel(voucherKernel).getBuyerOrderCosts(_tokenIdSupply);
+        deactivateConditionalCommit(_tokenIdSupply);
```

Comment:
It's supposed to be there, yes.
Functions `requestVoucherTKNTKNWithPermit` or `requestVoucherTKNTKNSameWithPermit` are called when user commits to buy voucher.
`deactivateConditionalCommit` checks if a given voucher set depends on some condition and if it does, it deactivates it on a gate contract to prevent buying the same voucher twice.

I will also comment that in the code.

Reply:
thanks !